### ISSUE TITLE
Pause wallpaper when desktop not visible

### ISSUE THREAD

**Issue by a-catgirl-dev** [rocksdanister/lively#1868]
**Is your feature request related to a problem? Please describe.**
When fullscreen applications are focused, the wallpaper pauses, but when a fullscreen application is covering the desktop, and you focus on an non-focused application, the wallpaper unpauses.

**Describe the solution you'd like**
When the desktop isn't visible, the wallpaper should stop. (obviously, there should be an option like 'wallpaper when desktop not visible' set to false)

**Describe alternatives you've considered**
No alternatives.

**Additional context**
tldr: pause wallpaper when the desktop isn't visible, regardless of applications focused

**Comment by FL4MIN** [comment]
I would love this feature as well, and have wallpapers paused whenever a full screen app is open (despite it is focused or not).

**Comment by litter-walker** [comment]
I share the same sentiment. I hope the author will consider this proposal.

**Comment by rocksdanister** [comment]
resolved: https://github.com/rocksdanister/lively/commit/034e9a6b4ca40c59caf6a6842667dd3494404f34
